performance/docker/build-push-1m-database-image.sh

```diff
@@ -30,24 +23,20 @@ echo "Loading 1m database into container..."
 docker run --name saraalert-1m-database -e MYSQL_ROOT_PASSWORD=root -d ghcr.io/saraalert/saraalert-1m-database:latest
 sleep 30
 docker exec -i saraalert-1m-database sh -c 'exec mysql -uroot -proot -e"CREATE DATABASE disease_trakker_development"'
-mysqldump --opt --column-statistics=0 --protocol=tcp --host=127.0.0.1 --user=root --password=root disease_trakker_development | docker exec -i saraalert-1m-database sh -c 'exec mysql -uroot -proot disease_trakker_development' 
+docker exec -i saraalert-1m-database sh -c 'exec mysql -uroot -proot disease_trakker_development' < $1
+echo "Showing table information"
+docker exec -i saraalert-1m-database sh -c 'exec mysql -uroot -proot disease_trakker_development -e "show tables;"'
+docker exec -i saraalert-1m-database sh -c 'exec mysql -uroot -proot disease_trakker_development -e "SELECT COUNT(*) FROM patients;"'
+docker exec -i saraalert-1m-database sh -c 'exec mysql -uroot -proot disease_trakker_development -e "SELECT COUNT(*) FROM symptoms;"'
 echo "Loaded 1m database into container."
 echo "Commiting new version of 1m database container..."
 docker commit saraalert-1m-database ghcr.io/saraalert/saraalert-1m-database:latest
 echo "Committed new version of 1m database container."
 docker stop saraalert-1m-database
 docker rm saraalert-1m-database
 
-# Find the oldest image
-# https://docs.github.com/en/rest/reference/packages
-OLDEST_CONTAINER_ID=$(curl -su $GHCR_USER:$GHCR_PASSWORD -H "Accept: application/vnd.github.v3+json" https://api.github.com/orgs/saraalert/packages/container/saraalert-1m-database/versions | bundle exec ruby -e 'require "json"; puts JSON.parse(ARGF.read).map { |d| d["id"] }.last')
-
 # Login to GitHub Container Registry (You only need to do this once)
 docker login -u $GHCR_USER -p $GHCR_PASSWORD ghcr.io
 
 # Push the new image
 docker push ghcr.io/saraalert/saraalert-1m-database:latest
-
-# Delete the oldest image
-# https://docs.github.com/en/rest/reference/packages
-curl -u $GHCR_USER:$GHCR_PASSWORD -X DELETE -H "Accept: application/vnd.github.v3+json" https://api.github.com/orgs/saraalert/packages/container/saraalert-1m-database/versions/$OLDEST_CONTAINER_ID
```

Comment:
Why isn't this necessary anymore? Was it ever necessary?

Reply:
I decided that it was a bad idea to automatically delete the previous image before we have time to verify that the new image works properly. Since we would be creating a new image so rarely I think it's fine if we delete the older images manually once we know that the new image works